lib/runtime/wasmer/exports_test.go

```diff
@@ -119,6 +121,56 @@ func TestInstance_Version_NodeRuntime(t *testing.T) {
 	require.Equal(t, expected, version.RuntimeVersion)
 }
 
+func balanceKey(t *testing.T, pub []byte) []byte {
```

Comment:
this function should be used by the build-spec command to turn the human-readable genesis into a raw genesis, but that could be done separately

Reply:
Created an issue for this. #1339